SpatialGDK/Source/Public/Interop/SpatialView.h

```diff
@@ -26,14 +28,19 @@ class SPATIALGDK_API USpatialView : public UObject
 
 	Worker_Authority GetAuthority(Worker_EntityId EntityId, Worker_ComponentId ComponentId);
 	SpatialUnrealMetadata* GetUnrealMetadata(Worker_EntityId EntityId);
+	SpatialEntityAcl* GetEntityACL(Worker_EntityId EntityId);
 
 private:
 	void OnAddComponent(const Worker_AddComponentOp& Op);
 	void OnRemoveEntity(const Worker_RemoveEntityOp& Op);
+	void OnComponentUpdate(const Worker_ComponentUpdateOp& Op);
 	void OnAuthorityChange(const Worker_AuthorityChangeOp& Op);
 
+	USpatialNetDriver* NetDriver;
 	USpatialReceiver* Receiver;
+	USpatialSender* Sender;
 
 	TMap<Worker_EntityId_Key, TMap<Worker_ComponentId, Worker_Authority>> EntityComponentAuthorityMap;
 	TMap<Worker_EntityId_Key, TSharedPtr<SpatialUnrealMetadata>> EntityUnrealMetadataMap;
+	TMap<Worker_EntityId_Key, TSharedPtr<SpatialEntityAcl>> EntityACLMap;
```

Comment:
We should do https://improbableio.atlassian.net/browse/UNR-561 sooner rather than later.